src/shogun/evaluation/StratifiedCrossValidationSplitting.cpp

```diff
@@ -145,12 +143,11 @@ void CStratifiedCrossValidationSplitting::build_subsets()
 			target_set%=m_subset_indices->get_num_elements();
 			SG_UNREF(next);
 		}
-
 		SG_UNREF(current);
```

Comment:
Oh, I forget to restore these lines in here. I will do it after the rest of the pr works good.